include/rbs/parser.h

```diff
@@ -27,9 +27,9 @@ typedef struct rbs_comment_t {
     rbs_position_t start;
     rbs_position_t end;
 
-    size_t line_size;
-    size_t line_count;
-    rbs_token_t *tokens;
+    size_t line_tokens_capacity;
+    size_t line_tokens_count;
+    rbs_token_t *line_tokens;
```

Comment:
This naming was confusing, because `line_size` sounds like "the size of a line".

What is actually means is "the capacity of the `line_tokens` buffer.